### PR TITLE
SCSS standardization

### PR DESCRIPTION
**Pull request by drummond-dds**
This PR adds in SCSS auto formatting so we can get our rules to be more consistent.

Its split into 3 commits: 

1. Add the command
2. Fix existing SCSS that is actually invalid
3. Run the formatting script

I can't work out if there is a `check` version of the script we can run. Is that needed?

### CODE REVIEW

**Comment by dandds** [comment]
@drummond-dds This looks good. Do you want to redo the last commit from master?

**Comment by drummond-dds** [comment]
@dandds I just pushed up

File: styles/components/_accordion_table.scss [hunk 1]
```diff
@@ -130,8 +130,7 @@
 
 #application-members {
   .accordion-table {
-    .accordion-table__head {
-    }
+    .accordion-table__head {}
```
Comment (drummond-dds) [inline styles/components/_accordion_table.scss]: This can be deleted

File: styles/components/_portfolio_layout.scss [hunk 2]
```diff
@@ -391,12 +400,12 @@
         max-width: 95%;
       }
 
-      .application-list-item__environment__name {
-      }
+      .application-list-item__environment__name {}
```
Comment (drummond-dds) [inline styles/components/_portfolio_layout.scss]: This can be deleted

File: styles/sections/_task_order.scss [hunk 3]
```diff
@@ -110,13 +111,14 @@
       dt {
         font-weight: bold;
       }
-      dd {
-      }
+
+      dd {}
```
Comment (drummond-dds) [inline styles/sections/_task_order.scss]: This can be deleted

**Comment by drummond-dds** [comment]
@dandds I fixed those small issues I identified

**Pull request merged**